js/src/hooks/useTargetAudienceFinalCountryCodes.js

```diff
@@ -30,16 +31,15 @@ import { STORE_KEY } from '.~/data';
  *
  */
 const useTargetAudienceFinalCountryCodes = () => {
-	return useSelect( ( select ) => {
-		const { getTargetAudience, getCountries, isResolving } = select(
-			STORE_KEY
-		);
+	const {
+		data: supportedCountries,
+		isResolving: countriesLoading,
```

Comment:
📝  The reason for still using `isResolving` is to keep the original state of its interface consistent.